x-pack/libbeat/management/managerV2.go

```diff
@@ -773,3 +871,32 @@ func didChange(previous map[string]*proto.UnitExpectedConfig, latest map[string]
 	}
 	return false
 }
+
+// cause returns the underlying cause of the error, if possible.
+// An error value has a cause if it implements the following
+// interface:
+//
+//	type causer interface {
+//	       Cause() error
+//	}
+//
+// If the error does not implement Cause, the original error will
+// be returned. If the error is nil, nil will be returned without further
+// investigation.
+// Source: https://github.com/pkg/errors/blob/614d223910a179a466c1767a985424175c39b465/errors.go#L264-L288
+func cause(err error) error {
```

Comment:
I don't think you need this, all uses of `Cause` were removed from Beats in https://github.com/elastic/beats/pull/31772

Reply:
> The Cause() function from github.com/pkg/errors is not compatible with the standard library's new error wrapping functionality.

> Specifically, Cause() will not call Unwrap() on a standard library wrapped error. Contrastingly, errors created or wrapped by github.com/pkg/errors implement Unwrap() (for example see [the source](https://github.com/pkg/errors/blob/5dd12d0cfe7f152f80558d591504ce685299311e/errors.go#L248)) and are compatible with the standard library.

Reply:
Done on 8eece56ece